vf_gltransition.c

```
@@ -280,6 +384,61 @@ static void setup_uniforms(AVFilterLink *fromLink)
   // TODO: initialize this in config_props for "to" input
   c->_toR = glGetUniformLocation(c->program, "_toR");
   glUniform1f(c->_toR, fromLink->w / (float)fromLink->h);
+
+  StringArray_t sa = parseQueryString(c->uniforms);
+  if(sa.len >0){
+    for(int i = 0;i<sa.len;i+=2){
+      GLint location = glGetUniformLocation(c->program, sa.strings[i]);
+      if (location >= 0) {
```

let's throw an error if this is not the case; better to be strict with potential errors here rather than ignoring it silently.

---

There is an err message `can't get location of uniform ..` if location is not valid, but I agree that throwing an error is the right choice.